### ISSUE TITLE
change ncurses package to provide either dynamic only or dynamic and static libraries

### ISSUE THREAD

**Issue by GitMensch** [msys2/MINGW-packages#3593]
The current [PKGBUILD](https://github.com/Alexpux/MINGW-packages/blob/master/mingw-w64-ncurses/PKGBUILD) passes `--without-shared` to the configure line and installs the static library as libncursesw.a + /lib/libncurses.a.
This obviously leads to bigger binaries...

I'd try to provide a PR but I'm unsure if we need the static library at all (= if both should be provided or only the dynamic one).

Opinions?

**Comment by mingwandroid** [comment]
We tend to provide both when possible.

**Comment by michaelsbradleyjr** [comment]
Why are both not provided for `mingw-w64-x86_64-ncurses` . Is it not possible or is it problematic to provide shared libraries for ncurses in addition to static libraries?

At present `mingw-w64-x86_64-ncurses` still provides only the static (`.a`) libraries.

**Comment by GitMensch** [comment]
@Alexpux It's been quite a long time since 6aef1eef909560c6ae8c077f09524d1d19dadb82 - Do you remember why you've added
`--without-shared --without-pthread`? Can we possibly get dir of those two flags or is there something/someone that speaks against this?

According to the [official MSYS2 statement from ncurses](https://github.com/mirror/ncurses/blob/45a2b5bc1a4536173178e9907feb268c708797c7/README.MinGW#L89) we actually should built with:

```
        	--without-libtool # replaced by --without-shared
        	--disable-home-terminfo
        	--enable-database
        	--disable-termcap # missing
        	--enable-sp-funcs
        	--enable-term-driver
        	--enable-interop
```

(and we have a bunch of other additional flags, while the additional flags mentioned in the paragraph after the above has a bunch of more flags.partially conflicting to those used in the MSYS2 package)

**Comment by MehdiChinoune** [comment]
Fixed by #11586
